roles/redpanda_broker/tasks/install-console-rpm.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install redpanda console (rpm)
+  ansible.builtin.package:
+    name:
+      - redpanda-console
+    state: "{{ redpanda_install_status }}"
+    update_cache: true
```

Comment:
Why the rpm task does not have to check for the ` https_proxy_value is defined and https_proxy_value | length > 0` `length==0` ?